wbooks/src/screens/book-detail/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+import BookDetail from './layout';
+import Books from '../../data/books.json';
+
+class BookDetailContainer extends Component {
+  componentWillMount() {
+    const id = this.props.match.params.number;
+    const item = Books.find(book => book.id.toString() === id);
+
+    this.setState({
+      item: item
```

Comment:
Usually we initially declare
```
state = { item: null }
```
to denote that there is an `item` that will be an object later

Reply:
Also another way is to `this.setState({ item });`